src/data/unleash/unleash.ts

```diff
@@ -5,12 +5,14 @@ import { ToggleNames, Toggles } from "@/data/unleash/unleash_types";
 export interface UnleashState {
   fetching: boolean;
   fetchingFailed: boolean;
+  triedFetchingToggles: boolean;
   toggles: Toggles;
 }
 
 export const initialState: UnleashState = {
   fetching: false,
   fetchingFailed: false,
+  triedFetchingToggles: false,
```

Comment:
Hva med å heller bruke  `fetchingCompleted`, `fetchinDone` eller `fetchingFinished`(kan velge og vrake her) som "standard" for at et kall er fullført?

Reply:
f.eks. fetchingDone er kanskje greiere ja (selv om man kan tro at det ble success da). Tenker uansett å ta i bruk react-query her etter hvert så da slipper vi finne på navn på disse flaggene 😃

Reply:
Ja, sant det 👍